src/lpcjobqueue/cluster.py

```diff
@@ -43,6 +43,7 @@ class LPCCondorJob(HTCondorJob):
     container_prefix = "/cvmfs/unpacked.cern.ch/registry.hub.docker.com/"
     config_name = "lpccondor"
     known_jobs = set()
+    env_name = os.path.basename(os.getenv('VIRTUAL_ENV', ''))
```

Comment:
Should the default for `os.getenv` be `.env` instead of `''` ?

Reply:
Good point! Updated in new commit